plugins.json

```diff
@@ -314,6 +314,14 @@
         "state": "development",
         "plugin_info": "https://github.com/pzarabadip/aiida-porousmaterials/blob/master/setup.json",
         "code_home": "https://github.com/pzarabadip/aiida-porousmaterials"
+    },
+    "plumed": {
+        "name": "aiida-plumed",
+        "entry_point": "plumed",
+        "state": "development",
+        "code_home": "https://github.com/ConradJohnston/aiida-plumed",
+        "plugin_info": "https://raw.github.com/ConradJohnston/aiida-plumed/AiiDA-v1.0-compatibility/setup.json",
+        "pip_url": "aiida-plumed",
+        "documentation_url": "http://aiida-plumed.readthedocs.io/"
```

Comment:
This doesn't seem to exist yet ;-)

Reply:
Fair cop. 
I've updated the status to "registered".

Reply:
ah, I was just referring to the documenation url.
It's fine to leave it blank if it does not exist.

From my point of view, "development" is the right status

Reply:
Changed the status back to "development"

Regarding the docs, the URL is reserved, but I'm having problems getting the (placeholder)  docs to build on ReadTheDocs.io.

Reply:
Ok, I see - in this case fine to merge.

In principle, if you follow the setup in https://github.com/aiidateam/aiida-diff it should work fine. If not, let me know.